README.md

```diff
@@ -76,6 +76,7 @@ COMMANDS:
   install-template                  Installs a project from templates
   build                             Build containers
   start-genesis                     Start containers from the genesis snapshot (erasing history) [aliases: start_genesis]
+  create-remote-genesis             Create genesis files to deploy on remote instances [aliases: create_remote_genesis]
```

Comment:
Could you also add a description of the genesis deploy process to the Deployment section below? Just something that shows step by step what a project needs to do to deploy their metagraph from genesis.